lib/elixir/lib/exception.ex

```diff
@@ -711,6 +711,67 @@ end
 
 defmodule ArithmeticError do
   defexception message: "bad argument in arithmetic expression"
+
+  @impl true
+  def blame(
+        %{message: default_message} = exception,
+        [{:erlang, func, args, _} | _] = stacktrace
+      ) do
+    message =
+      default_message <>
+        case {func, args} do
+          {:+, [a]} ->
+            ": tried to sum #{inspect(a)}"
+
+          {:+, [a, b]} ->
+            ": tried to sum #{inspect(a)} and #{inspect(b)}"
+
+          {:-, [a]} ->
+            ": tried to substract #{inspect(a)}"
+
+          {:-, [a, b]} ->
+            ": tried to substract #{inspect(a)} and #{inspect(b)}"
+
+          {:*, [a, b]} ->
+            ": tried to multiply #{inspect(a)} by #{inspect(b)}"
+
+          {:/, [a, b]} ->
+            ": tried to divide #{inspect(a)} by #{inspect(b)}"
+
+          {:div, [a, b]} ->
+            ": tried to divide #{inspect(a)} by #{inspect(b)}"
```

Comment:
since we have two divisions, what if we say "divide (div)" here and "divide (/)" above?